source/DiscordManager.hx

```diff
@@ -0,0 +1,57 @@
+package ;
+
+import flixel.util.FlxTimer;
+import discord_rpc.DiscordRpc;
+
+class DiscordManager
+{
+    public static function init() {
+        DiscordRpc.start({
+            clientID : "806658783132516383",
+            onReady : _onReady,
+            onError : _onError,
+            onDisconnected : _onDisconnected
+        });
+
+        DiscordRpc.process();
+        new FlxTimer().start(1, process, 0);
```

Comment:
At first I had concerns this could impact performance, but after some testing this does not seem to affect performance at all.